packages/explorer/src/queries/dozer/useDeployedTablesQuery.ts

```diff
@@ -0,0 +1,50 @@
+import { useParams } from "next/navigation";
+import { Hex } from "viem";
+import { useQuery } from "@tanstack/react-query";
+import { decodeTable } from "../../app/(explorer)/[chainName]/worlds/[worldAddress]/explore/utils/decodeTable";
+import { internalMUDNamespaces } from "../../common";
+import { useDozerUrl } from "../../hooks/useDozerUrl";
+
+export type DozerResponse = {
+  block_height: number;
+  result: [Hex[][]];
+};
+
+export function useDeployedTablesQuery() {
+  const { worldAddress, chainName } = useParams();
+  const dozerUrl = useDozerUrl();
+
+  return useQuery({
+    queryKey: ["deployedTables", worldAddress, chainName],
+    queryFn: async () => {
+      const response = await fetch(dozerUrl, {
```

Comment:
maybe could use the helper for this?
https://github.com/latticexyz/mud/blob/main/packages/store-sync/src/sql/fetchRecords.ts

Reply:
oh this looks really useful, checking